{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2019-16227",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "An issue was discovered in py-lmdb 0.97. For certain values of mn_flags, mdb_cursor_set triggers a memcpy with an invalid write operation within mdb_xcursor_init1. NOTE: this outcome occurs when accessing a data.mdb file supplied by an attacker."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20memcpy%20illegal%20dst",
                "refsource": "MISC",
                "url": "https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20memcpy%20illegal%20dst"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}